src/CoreBundle/Bridge/Chameleon/Module/Sidebar/SidebarBackendModule.php

```diff
@@ -0,0 +1,166 @@
+{
+    /**
+     * @var LanguageServiceInterface
+     */
+    private $languageService;
+
+    public function __construct(LanguageServiceInterface $languageService)
+    {
+        parent::__construct();
+        $this->languageService = $languageService;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function Accept(\IMapperVisitorRestricted $oVisitor, $bCachingEnabled, \IMapperCacheTriggerRestricted $oCacheTriggerManager)
+    {
+        $oVisitor->SetMappedValue('menuItems', $this->getMenuItems());
+    }
+
+    private function getMenuItems(): array
+    {
+        $activeLanguageId = $this->languageService->getActiveLanguageId();
+        $activeUser = \TCMSUser::GetActiveUser();
+        if (null === $activeUser) {
+            return [];
+        }
+
+        $menuItemsRaw = [];
+        $this->addTableMenuItems($menuItemsRaw, $activeUser, $activeLanguageId);
+        $this->addModuleMenuItems($menuItemsRaw, $activeUser, $activeLanguageId);
+
+        return $this->mergeMenuItems($menuItemsRaw);
+    }
+
+    private function addTableMenuItems(array &$aMenuItemsTemp, \TCMSUser $activeUser, string $activeLanguageId): void
+    {
+        $query = "SELECT * FROM `cms_tbl_conf` WHERE `cms_content_box_id` <> '' AND `cms_content_box_id` <> '0'";
+        $tableList = \TdbCmsTblConfList::GetList($query, $activeLanguageId);
+        while ($tableObject = $tableList->Next()) {
+            if (false === $this->isTableAccessAllowed($activeUser, $tableObject)) {
+                continue;
+            }
+            if (false === \array_key_exists($tableObject->fieldCmsContentBoxId, $aMenuItemsTemp)) {
+                $aMenuItemsTemp[$tableObject->fieldCmsContentBoxId] = [];
+            }
+            $aMenuItemsTemp[$tableObject->fieldCmsContentBoxId][] = new MenuItem(
+                $tableObject->fieldTranslation,
+                $tableObject->fieldIconList,
+                $this->getTableTargetUrl($tableObject->id)
+            );
+        }
+    }
+
+    private function isTableAccessAllowed(\TCMSUser $activeUser, \TdbCmsTblConf $tableObject): bool
+    {
+        $tableInUserGroup = $activeUser->oAccessManager->user->IsInGroups($tableObject->fieldCmsUsergroupId);
+        $isEditAllowed = $activeUser->oAccessManager->HasEditPermission($tableObject->fieldName);
+        $isShowAllReadonlyAllowed = $activeUser->oAccessManager->HasShowAllReadOnlyPermission($tableObject->fieldName);
+
+        return (true === $tableInUserGroup && (true === $isEditAllowed || true === $isShowAllReadonlyAllowed));
+    }
+
+    private function getTableTargetUrl(string $tableId): string
+    {
+        return PATH_CMS_CONTROLLER."?pagedef=tablemanager&id=$tableId";
+    }
+
+    private function addModuleMenuItems(array &$menuItemsRaw, \TCMSUser $activeUser, string $activeLanguageId): void
+    {
+        $query = "SELECT * FROM `cms_module` WHERE `active` = '1'";
+        $cmsModuleList = \TdbCmsModuleList::GetList($query, $activeLanguageId);
```

Comment:
As above... any order by will be needed. Problem is that we have 2 sources of elements and can't order them combined using a backend function.